POS_Image-Graphical7/graphical-7.0.0/root/etc/systemd/system/install-local-bootloader.service

```diff
@@ -13,7 +13,7 @@ Type=oneshot
 ExecStartPre=/bin/bash -c 'if [ -e /sys/firmware/efi ]; then sed -i -e "s|^LOADER_TYPE=.*|LOADER_TYPE=\\"grub2-efi\\"|" /etc/sysconfig/bootloader; fi'
 ExecStartPre=/bin/bash -c 'if [ ! -e /sys/firmware/efi ]; then sed -i -e "s|^LOADER_TYPE=.*|LOADER_TYPE=\\"grub2\\"|" /etc/sysconfig/bootloader; fi'
 ExecStart=/bin/bash -c 'pbl --install ; dracut -f ; pbl --config '
```

Comment:
Is there no problem with `pbl` and `dracut` using relatives?

Reply:
This is parsed by bash so it works. Only systemd has problems with relative paths.